src/root.js

```diff
@@ -12,7 +12,10 @@ import { AuthProvider } from 'contexts/useAuthContext'
 import { FlashMessageProvider } from 'contexts/useFlashMessageContext'
 import HFScreenWidthContext from 'holofuel/contexts/screenWidth'
 import { FlashMessageProvider as HFFlashMessageProvider } from 'holofuel/contexts/useFlashMessageContext'
-import AcceptRequestedOffers from './holofuel/AcceptRequestedOffers'
+import { CurrentUserProvider as HFCurrentUserProvider } from 'holofuel/contexts/useCurrentUserContext'
+import AcceptRequestedOffers from 'holofuel/components/wrappers/AcceptRequestedOffers'
+import LoadCurrentUser from 'holofuel/components/wrappers/LoadCurrentUser'
+
 import PromptForNickname from './holofuel/PromptForNickname'
```

Comment:
I would like to include this in the wrapper folder as well. (This is non-blocking though.)

Reply:
Ah yes, good catch